gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Apr 14 20:53:26 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
```

Comment:
I'd like to continue using the `bin` variant (it's half the size)

Reply:
Sure, I changed it to the `bin` variant.